payment_manual/indico_payment_manual/translations/de_DE/LC_MESSAGES/messages-js.po

```diff
@@ -0,0 +1,23 @@
+# Translations template for PROJECT.
+# Copyright (C) 2015 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+#
+# Translators:
+# Thomas Baron <thomas.baron@cern.ch>, 2015
+msgid ""
+msgstr ""
+"Project-Id-Version: Indico\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2017-06-26 09:51+0200\n"
+"PO-Revision-Date: 2017-06-26 09:51+0200\n"
+"Last-Translator: Dennis Riehle <webmaster@riehle-web.com>\n"
+"Language-Team: German (Germany) (http://www.transifex.com/projects/p/indico/language/de_DE/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+"Language: de_DE\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+msgid "Indico"
```

Comment:
I really wonder where this is coming from... A lone "Indico" should never be marked for translation (and I couldn't find it in the plugin either). FWIW, the plugin doesn't even have any JavaScript...